frontend/src/pages/organization/SecretManagerOverviewPage/components/AllProjectView.tsx

```diff
@@ -0,0 +1,286 @@
+import { useState } from "react";
```

Comment:
can we add handling so that when project admin is not logged in and they click on a project access request email, it would prompt them to login to Infisical and then redirect them to the "grant access" modal? 

we already do this for authenticated secret shares... not sure though how straightforward it would be here though
<img width="984" alt="image" src="https://github.com/user-attachments/assets/5bb7ad7d-e36b-443d-96f4-aa1498905fea" />

Reply:
Checking

Reply:
The issue with this is - we need modify our auth middleware in frontend. 

I think this can be converted to a general behavior where we redirect user to auth and then back to login. But I think it's better to do it in another ticket. This is because we have a lot of moving parts in login in frontend.

Reply:
copy that